portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsPreview.testcase

```diff
@@ -427,14 +427,44 @@ definition {
 	}
 
 	@description = "Verify that translations can be previewed"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test TranslationsCanBePreviewed {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133781 TranslationsCanBePreviewed pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Text");
 
+		Form.addTranslation(localizedLanguage = "Spanish (Spain)");
+
+		Form.editElementSetFieldLabel(fieldValue = "Texto");
```

Comment:
@MoryAlvesBR, you should use the macro to change the Field Label and not the Element Set Field Label, can you please fix it?